routemaster/client.rb

```diff
@@ -99,14 +133,22 @@ def monitor_topics
         raise 'failed to connect to /topics'
       end
 
-      JSON(response.body).map do |raw_topic|
+      Oj.load(response.body).map do |raw_topic|
         Topic.new raw_topic
       end
     end
 
 
     private
 
+    def _stringify_keys(hash)
+      hash.dup.tap do |h|
+        h.keys.each do |k|
```

Comment:
This is private, so we know the use cases are limited to this class, so we know the hash is always flat (because the API spec says so).